src/Pos/WorkMode.hs

```diff
@@ -72,6 +73,7 @@ modeContext [d|
         !(PeerStateTag  ::: PeerStateCtx Production)
         !(JsonLogConfig ::: JsonLogConfig)
         !(LoggerName    ::: LoggerName)
+        !(RelayContext  ::: RelayContext Production)
```

Comment:
Yes, it's not used at this point and can be eliminated, esp. given we need to hardcode it this way